atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolIntegrationTest.java

```diff
@@ -74,14 +75,34 @@ public void testTokenMapping() {
             clientPool.getRandomHostForKey("A".getBytes(StandardCharsets.UTF_8));
 
             if (tokenRange.hasLowerBound()) {
-                assertTrue(hosts.contains(clientPool.getRandomHostForKey(tokenRange.lowerEndpoint().bytes)));
+                assertThat(hosts).contains(clientPool.getRandomHostForKey(tokenRange.lowerEndpoint().bytes));
             }
             if (tokenRange.hasUpperBound()) {
-                assertTrue(hosts.contains(clientPool.getRandomHostForKey(tokenRange.upperEndpoint().bytes)));
+                assertThat(hosts).contains(clientPool.getRandomHostForKey(tokenRange.upperEndpoint().bytes));
             }
         }
     }
 
+    @Test
+    public void testRefreshLocalHosts() {
+        HostLocation localLocation = HostLocation.of("dc1", "rack1");
+        HostLocation remoteLocation = HostLocation.of("dc1", "rack4");
+
+        assertThat(getLocalHostsUsingLocation(localLocation)).hasSize(1);
+        assertThat(getLocalHostsUsingLocation(remoteLocation)).hasSize(0);
```

Comment:
Your asserts should describe the exact condition you want. So here, you want `.isEmpty()` and `.containsExactly(someLocalHost)`